packages/atomic/src/components/commerce/product-template-components/atomic-product-price/atomic-product-price.new.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import {wrapInCommerceInterface} from '@coveo/atomic-storybook-utils/commerce/commerce-interface-wrapper';
+import {wrapInCommerceProductList} from '@coveo/atomic-storybook-utils/commerce/commerce-product-list-wrapper';
+import {wrapInProductTemplate} from '@coveo/atomic-storybook-utils/commerce/commerce-product-template-wrapper';
+import {parameters} from '@coveo/atomic-storybook-utils/common/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic-storybook-utils/common/render-component';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+
+const {
+  decorator: commerceInterfaceDecorator,
+  play: initializeCommerceInterface,
+} = wrapInCommerceInterface({
+  skipFirstSearch: false,
+  type: 'product-listing',
+  engineConfig: {
+    context: {
+      view: {
+        url: 'https://sports.barca.group/browse/promotions/ui-kit-testing-product-price',
```

Comment:
I made this listing page for my specific testing needs for this component.

Reply:
I think this is a fine approach to ensure that we have something stable for storybook and testing.

I did something similar for the product-multi-value-text component.

This might be a good time to think about standardizing the way we identify these PLPs in our org.

Now that I think of it, we don't really need the "sports.barca.group/browse/promotions" part in the URL. For instance, we could use something like `https://ui-kit/atomic/storybook/atomic-product-price` (this is just an example, I haven't given this much thought).

Probably worth discussing this in person.